crates/sys/src/inlines/furi_hal_gpio.rs

```diff
@@ -37,6 +53,12 @@ pub unsafe extern "C" fn furi_hal_gpio_read(gpio: *const sys::GpioPin) -> bool {
 }
 
 /// GPIO read pin.
+///
+/// # Safety
+///
+/// Safety invariants are not yet defined for GPIO.
+///
```

Comment:
We've defined the safety invariants we require for this Rust code. Any other requirements relate to the correctness of the inline method's API as called by the SDK.
```suggestion
```